cpp/bench/ann/src/common/benchmark.hpp

```diff
@@ -571,7 +571,7 @@ void dispatch_benchmark(std::string cmdline,
       }
     }
     std::swap(more_indices, indices);  // update the config in case algorithms need to access it
-    register_build<T>(dataset, more_indices, force_overwrite, no_lap_sync);
+    register_build<T>(dataset, indices, force_overwrite, no_lap_sync);
```

Comment:
Thanks for catching this bug I introduced yesterday in https://github.com/rapidsai/cuvs/pull/647